models.py

```diff
@@ -146,13 +163,19 @@ def delete_custom_model(self):
         return self.STT.delete_language_model(self.get_customization_id())
 
     def train_custom_model(self):
-        return self.STT.train_language_model(self.get_customization_id())
+        resp = self.STT.train_language_model(self.get_customization_id())
+        self.wait_until('available', "Training")
+        print("Training complete. Custom model is ready to use.")
```

Comment:
eprint please (uses stderr), allows redirect of stdout to file

Reply:
change made